data/scripts/talkactions/commands/attributes.lua

```diff
@@ -0,0 +1,322 @@
+local talk = TalkAction("/attr")
+
+local itemFunctions = {
+    ["actionid"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setActionId(target)
+        end
+    },
+
+    ["uniqueid"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setAttribute(ITEM_ATTRIBUTE_UNIQUEID, target)
+        end
+    },
+
+    ["description"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setAttribute(ITEM_ATTRIBUTE_DESCRIPTION, target)
+        end
+    },
+
+    ["name"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setAttribute(ITEM_ATTRIBUTE_NAME, target)
+        end
+    },
+
+    ["remove"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:remove()
+        end
+    },
+
+    ["decay"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:decay()
+        end
+    },
+
+    ["transform"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:transform(target)
+        end
+    },
+
+    ["clone"] = {
```

Comment:
Why clone if the result obtained is not stored anywhere and we cannot use the cloned article, this code must be improved if we really want this clone option, as it currently  is useless